Functions/Administration/xASL_adm_DefineASLSequence.m

```diff
@@ -5,87 +5,86 @@
 % FORMAT: [xQ] = xASL_adm_DefineASLSequence(xQ, bVerbose)
 %
 % INPUT:
-%   xQ                - x.Q structure containing all input parameters (REQUIRED)
-%   xQ.readoutDim   - dimensionality of readout (2D or 3D) (OPTIONAL)
-%   xQ.Vendor       - Either 'GE', 'Philips', 'Siemens' (OPTIONAL)
-%   bVerbose         - verbose output (OPTIONAL, DEFAULT=true)
+%   xQ                   - x.Q structure containing all input parameters (REQUIRED)
+%   xQ.MRAcquisitionType - dimensionality of readout (2D or 3D) (OPTIONAL)
+%   xQ.Vendor            - Either 'GE', 'Philips', 'Siemens' (OPTIONAL)
+%   bVerbose             - verbose output (OPTIONAL, DEFAULT=true)
 %
 % OUTPUT:
-%   xQ               - x structure containing all output parameters
-%   xQ.Sequence    - sequence type (readout)
+%   xQ                   - x structure containing all output parameters
+%   xQ.PulseSequenceType - sequence type (readout)
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
 % DESCRIPTION: This ExploreASL function tries to check what ASL sequence is
-% being processed, if this was not already defined in xQ.Sequence.
+% being processed, if this was not already defined in xQ.PulseSequenceType.
 % It does so by checking known combinations of readout dimensionality
-% (xQ.readoutDim) and Vendor, knowing the product sequences of the Vendors.
+% (xQ.MRAcquisitionType) and Vendor, knowing the product sequences of the Vendors.
 %
 % EXAMPLE: xQ = xASL_adm_DefineASLSequence(xQ);
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
 % Copyright 2015-2024 ExploreASL
 
 
-%% Check quantification fields readoutDim & xQ.Sequence
+%% Check quantification fields MRAcquisitionType & xQ.PulseSequenceType
 if nargin<2 || isempty(bVerbose)
     bVerbose = true;
 end
 
-if ~isfield(xQ, 'readoutDim') || isempty(xQ.readoutDim)
-    warning('xQ.readoutDim parameter missing');
+if ~isfield(xQ, 'MRAcquisitionType') || isempty(xQ.MRAcquisitionType)
+    warning('xQ.MRAcquisitionType parameter missing');
 end
 
 % Check for illegal sequence definitions
```

Comment:
This was done for Sequence which is MRAcquisitionType _ PulseSequenceType. So now we only do this for PulseSequenceType, let's also do this for MRAcquisitionType.